src/test/scala/cromwell/engine/backend/jes/JesBackendSpec.scala

```diff
@@ -1,13 +1,14 @@
 package cromwell.engine.backend.jes
 
+import java.net.URL
+
 import cromwell.binding.values.{WdlFile, WdlString}
 import cromwell.binding.{Call, CallInputs}
 import cromwell.engine.workflow.WorkflowOptions
-import org.scalatest.mock.MockitoSugar
 import org.scalatest.{FlatSpec, Matchers}
-import spray.json.JsObject
+import org.specs2.mock.Mockito
```

Comment:
I thought this might have been a mistake, glad to hear that's not the case.  :smile:  I was also a bit surprised to see we even had specs2 as a dependency, but `sbt test:dependency-tree` tells me Akka TestKit is pulling this in.

These are kinda cool, but I'm not sure how much they're buying us here.  Wouldn't we get essentially equivalent behavior with

```scala
val anyString = ""
val anyUrl: URL = null
```